common/src/main/java/com/wynntils/services/statistics/StatisticsCollectors.java

```diff
@@ -63,53 +64,55 @@ public void onMythicFoundEvent(MythicFoundEvent event) {
 
     @SubscribeEvent
     public void onRaidCompleted(RaidEndedEvent.Completed event) {
-        switch (event.getRaid()) {
-            case NEST_OF_THE_GROOTSLANGS -> {
-                Services.Statistics.increaseStatistics(StatisticKind.NEST_OF_THE_GROOTSLANGS_SUCCEEDED);
-                Services.Statistics.addToStatistics(
-                        StatisticKind.NEST_OF_THE_GROOTSLANGS_TIME_ELAPSED, event.getRaidTime() / 1000);
-            }
-            case ORPHIONS_NEXUS_OF_LIGHT -> {
-                Services.Statistics.increaseStatistics(StatisticKind.ORPHIONS_NEXUS_OF_LIGHT_SUCCEEDED);
-                Services.Statistics.addToStatistics(
-                        StatisticKind.ORPHIONS_NEXUS_OF_LIGHT_TIME_ELAPSED, event.getRaidTime() / 1000);
-            }
-            case THE_CANYON_COLOSSUS -> {
-                Services.Statistics.increaseStatistics(StatisticKind.THE_CANYON_COLOSSUS_SUCCEEDED);
-                Services.Statistics.addToStatistics(
-                        StatisticKind.THE_CANYON_COLOSSUS_TIME_ELAPSED, event.getRaidTime() / 1000);
-            }
-            case THE_NAMELESS_ANOMALY -> {
-                Services.Statistics.increaseStatistics(StatisticKind.THE_NAMELESS_ANOMALY_SUCCEEDED);
-                Services.Statistics.addToStatistics(
-                        StatisticKind.THE_NAMELESS_ANOMALY_TIME_ELAPSED, event.getRaidTime() / 1000);
-            }
+        RaidKind raidKind = event.getRaid().getRaidKind();
+
+        if (raidKind.equals(Models.Raid.NEST_OF_THE_GROOTSLANG)) {
```

Comment:
I know this was my (perhaps bad) suggestion, but what if we did instanceof checks on RaidKind? That would bring the API closer to how containers are implemented currently, and we could get rid of the static fields. I should have suggested this the first time around anyways.